src/ir_Argo.cpp

```diff
@@ -63,6 +63,28 @@ void IRArgoAC::begin(void) { _irsend.begin(); }
 void IRArgoAC::send(const uint16_t repeat) {
   _irsend.sendArgo(getRaw(), kArgoStateLength, repeat);
 }
+
+/// Send current room temperature for the iFeel feature as a silent IR
+/// message (no acknowledgement from the device).
+/// @param[in] temp The temperature in degrees celsius.
+/// @param[in] repeat Nr. of times the message will be repeated.
+void IRArgoAC::sendTemperature(const uint8_t temp, const uint16_t repeat) {
+    unsigned char tempc = temp - kArgoTempDelta;
+    unsigned char check = 52 + tempc;
+    unsigned char end = 0b011;
```

Comment:
Please use `uint8_t` instead of `unsigned char`.
See: https://github.com/crankyoldgit/IRremoteESP8266/wiki/Library-Maintainers-Guide#c